partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -48,11 +48,11 @@
       "The schema generator to use with partitioners.";
   public static final String SCHEMA_GENERATOR_CLASS_DISPLAY = "Schema Generator Class";
 
-  public static final String PARTITION_FIELD_NAME_CONFIG = "partition.field.name";
```

Comment:
I think we'd need to retain both properties for a while for the sake of backwards compatibility.